### DISCUSSION TITLE
How to remove the green line in search results

### DISCUSSION THREAD

**Discussion by Skyxim** [darkreader/darkreader#7064] [Q&A]
It’s the first time I use it. I don’t know how to set it. Google, bing, baidu have this kind of green line.
I want to remove him, do I need to modify the css?
![image](https://user-images.githubusercontent.com/70276018/136953118-48e51e61-b69c-4deb-9744-a58497892030.png)
![image](https://user-images.githubusercontent.com/70276018/136953363-675dd5ba-6092-455d-ae8e-d13d03f48185.png)

**Comment by Gusted** [comment]
That's pretty weird such line shouldn't be their in the first place, do you have any other extensions other than Dark Reader?

**Reply by Skyxim** [reply]
Thanks for answering, I found out that another plugin conflicted with me and modified the configuration of the next one.
